tools/update-libraries.js

```diff
@@ -0,0 +1,88 @@
+#!/usr/bin/env node
+'use strict';
+
+const fs = require('fs-extra');
+const path = require('path');
+
+const root = path.join(__dirname, '..');
+
+const files = {
+  'codemirror': [
+    'addon/comment/comment.js',
+    'addon/dialog',
```

Comment:
er, I meant the script should update the currently existing files instead of hardcoding the list so that we don't have to modify this script when we add a codemirror thingy. Consequently, there should be no need for the beautiful diagram in the readme file.

Reply:
That sounds like a good idea.